scipy/optimize/tests/test_linprog.py

```diff
@@ -1675,6 +1676,68 @@ def test_crossover(self):
         # there should be nonzero crossover iterations for IPM (only)
         assert_equal(res.crossover_nit == 0, self.method != "highs-ipm")
 
+    def test_marginals(self):
```

Comment:
Confirmed that this is equivalent to what we added in mckib2/scipy#10 [here](https://github.com/mckib2/scipy/pull/10/commits/07552b7f013956807cf4a6fa6881ca170a3fefc1).